packages/web-components/fast-foundation/src/patterns/start-end.ts

```diff
@@ -41,29 +40,20 @@ export class StartEnd {
  *
  * @public
  */
-export const endSlotTemplate: (
-    context: ElementDefinitionContext,
-    definition: EndOptions
-) => ViewTemplate<StartEnd> = (
-    context: ElementDefinitionContext,
-    definition: EndOptions
-) => html`
-    <slot name="end" ${ref("end")}>${definition.end || ""}</slot>
-`;
+export function createEndSlotTemplate(options: EndOptions): ViewTemplate<StartEnd> {
+    return html`
+        <slot name="end" ${ref("end")}>${options.end || ""}</slot>
+    `;
+}
```

Comment:
Here's an example where I'm not sure the `create` makes a ton of sense with a somewhat static template, but again - if we're going to transition this to the proposed "builder" approach, then I think that terminology will make sense. If it always returns a fixed ViewTemplate though, it feels like a bit of a semantic overload on "create".

Reply:
I don't know that we'll have a need for this particular helper at all once we have the builders. In my prototype I built this pattern into the anatomy base class so it could be consistently applied. In this case, I just went for a consistent naming pattern across functions that create templates.